src/types/index.d.ts

```diff
@@ -0,0 +1,139 @@
+declare type actionsInterface = import('./actionsInterface').Actions;
+declare type actionTypesInterface = import('./actionTypesInterface').actionTypes;
+
+declare module 'gisida' {
+
+    interface generalFunc {
+        (...args: any[]):any
+    }
+
+    interface generalObjects {
+        [key: string]: any
+    }
```

Comment:
Can we import/use `FlexObject` definitions from one of the js-tools packages?

Reply:
@cKellyDesign - Have realized the  `FlexObject` and normal `object` have same interface so have just used normal `object`